examples/hybrid/plane/bubble_2d_invariant_rhoe.jl

```diff
@@ -215,7 +220,8 @@ function rhs_invariant!(dY, Y, _, t)
     # 3) total energy
 
     @. dρe -= hdiv(cuw * (cρe + cp))
-    @. dρe -= vdivf2c(fw * Ic2f(cρe + cp))
+    @. dρe -=
+        vdivf2c((Ic2f(cρ) * third_order_upwind_c2f.(fw, (cρe + cp) ./ cρ)))
```

Comment:
`third_order_upwind_c2f.(fw, (cρe + cp) ./ cρ)))`
is the explicit dot operator necessary ? **@.** d\rhoe = <RHS> already assigns the broadcast rule right ? This change is otherwise OK.

Reply:
Thanks Akshay for the prompt review. Removed the unnecessary dots and rebased. Ready to merge on my end. Thank you